app.js

```diff
@@ -33,16 +32,18 @@ app.use(
   }),
 );
 
+
 app.use(cookieParser());
 
 app.use('/users', userRouter);
 app.use('/auth', authRouter);
 // TODO remove
 app.use('/test', [verifyToken, userRouter]);
 
+
 app.use(sectionSegmentRouter);
 app.use(monitorLogRouter);
-
+app.use('/users', usersRouter);
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'usersRouter' is not defined.